index.html

```diff
@@ -4,9 +4,54 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My-portfolio</title>
+    <title>MyPortfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inconsolata:wght@400;500;600;700&family=Poppins:ital,wght@1,300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inconsolata:wght@400;500;600;700&family=Inter:wght@200;300;400;500;600&family=Poppins:ital,wght@1,300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/86cc835e69.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    <h1>My-portfolio</h1>
+    <header>
+        <div class="navi container">
+            <div class="logo">
+                <figure>
+                    <img src="assets/bw.png" alt="sasukeR">
+                    <figcaption>sasukeR</figcaption>
+                </figure>
+            </div>
+            <nav class="panel">
+                <a href="#">Home</a>
+                <a href="#">About</a>
+                <a href="#">Portfolio</a>
+                <a href="#">Contact Me</a>
+            </nav>
```

Comment:
Hi dude! good job
I suggest you use tag li for each a.
Because if use li you can improve the accessibility whit TAB key.
or use the tag unoredered list (ul).